tests/test_readonly_store_decorator.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# coding=utf8
+
+from simplekv.decorator import ReadOnlyStore
+from simplekv.fs import FilesystemStore
+import pytest
+
+
+class TestReadOnlyStoreDecorator(object):
+    def test_readonly(self, tmpdir):
+        store0 = FilesystemStore(tmpdir)
```

Comment:
It's probably a good idea to use a `DictStore` here, for speed. I know my implementation used a `FilesystemStore`, but this would be an improvement, I think.